### PR TITLE
Upgrade to cookiecutter v13

### PR DESCRIPTION
**Pull request by brianhelba**

### CODE REVIEW

**Comment by brianhelba** [comment]
Please merge once this is approved.

File: optimal_transport_morphometry/core/rest/image.py
Comment (zachmullen) [inline optimal_transport_morphometry/core/rest/image.py:5]: @brianhelba I haven't kept up here, what's the story on drf_yasg?

**Review by zachmullen — APPROVED** [review]

File: optimal_transport_morphometry/core/rest/image.py
Comment (brianhelba) [reply optimal_transport_morphometry/core/rest/image.py:5]: @zachmullen It was unmaintained, then forked as `drf_yasg2`. About a week after the fork, the original maintainer showed up again, and it was agreed by all parties to unfork.

Just use `drf_yasg` in the future. They are accepting PRs if we find issues.

**Pull request merged**
